send-message/README.md

```diff
@@ -0,0 +1,66 @@
+# Zulip Send Message Action
+
+This action sends a message to [Zulip](https://zulip.com/).
+
+## Inputs
+
+### `api-key`
+
+**Required** [API key](https://zulip.com/api/api-keys) used to interact with the Zulip API. You can get an API key through Zulip's web interface.
+
+### `email`
+
+**Required** Email address of the user who owns the API key mentioned above.
+
+### `organization-url`
+
+**Required** Zulip organization canonical URL.
+
+### `to`
+
+**Required** For stream messages, either the name or integer ID of the stream.
+For private messages, either a list containing integer user IDs or a list containing string email addresses.
+
+### `type`
+
+**Required** The type of message to be sent. `private` for a private message and `stream` for a stream message.
+Must be one of: `private`, `stream`.
+
+### `topic`
+
+**Optional** The topic of the message. Only required for stream messages (`type="stream"`), ignored otherwise.
+Maximum length of 60 characters.
+
+### `content`
+
+**Required** The content of the message. Maximum message size of 10000 bytes.
+Format your message using [Zulip Markdown](https://zulip.com/help/format-your-message-using-markdown).
+
+## Example usage
+
+**Send a stream message**
+```yml
+- name: Send a stream message
+  uses: zulip/github-actions-zulip/send-message@v1.0
```

Comment:
I think the next release should be `v1.0.0` and we should recommend using `v1.0` (it will use the latest bug fixes release in the 1.0 series)

Reply:
Sounds good!  Is `@v1.0` the usual naming scheme for how to indicate versions?  (Or is `@1.0` more normal)?

Reply:
There’s no magic that matches `v1.0` to `v1.0.0`; that’s implemented by literally adding an extra tag for the same commit (see https://github.com/actions/checkout/tags, for example).

Since semver minor bumps are supposed to be backwards compatible, there should be no reason to have a `v1.0` that doesn’t get bumped to match `v1.1.0`. Our extra tag should be simply `v1`.

Reply:
> Is @v1.0 the usual naming scheme for how to indicate versions? 

`v1` is the recommended naming scheme: https://docs.github.com/en/actions/creating-actions/about-actions#using-release-management-for-actions

> There’s no magic that matches v1.0 to v1.0.0; that’s implemented by literally adding an extra tag for the same commit

Oh you're right, I thought it was a built-in feature of GitHub Action (based on semver).

For reference, the CodeQL Action is using a branch named `v1`: https://github.com/github/codeql-action/tree/v1 (instead of moving the major version tag to point to the Git ref of the current release)

> Since semver minor bumps are supposed to be backwards compatible, there should be no reason to have a v1.0 that doesn’t get bumped to match v1.1.0. Our extra tag should be simply v1.

Agreed, we should use `v1` not `v1.0` 👍🏻